src/AcmTable/AcmTable.tsx

```diff
@@ -720,6 +731,14 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
                                     /* istanbul ignore next */
                                     rows.length && (bulkActions?.length || !!props.onSelect) ? onSelect : undefined
                                 }
+                                onCollapse={
+                                    props.addSubRows
+                                        ? (_event, rowIndex, isOpen) => {
+                                              const rowKey = rows[rowIndex].props.key.toString()
+                                              setExpanded({ ...expanded, [rowKey]: isOpen })
+                                          }
+                                        : undefined
+                                }
```

Comment:
I guess grouping and expandable rows will either need to be completely mutually exclusive, or you can mix as long as any 1 row only supports 1 of these features. If the latter, then we'll need to decide which takes precedence.

Reply:
I think we should have some more discussions about the grouping thing too, I think the design for the app-ui table is too complex with all the different grouping and sorting behaviors.  But yeah, it might be easier to have both mechanisms separate, I don't think there will be a use case where both would be used though 🤔